staging/vhost-device-gpu/src/vhu_gpu.rs

```diff
@@ -588,32 +598,19 @@ impl VhostUserBackendMut for VhostUserGpuBackend {
         if evset != EventSet::IN {
             return Err(Error::HandleEventNotEpollIn.into());
         };
-
-        match device_event {
-            CONTROL_QUEUE => {
```

Comment:
I would prefer keeping the match statement here instead of in the `handle_queue_event`.  This would IMO be a more clear separation, where we handle the invalid `device_event` in the "entry point" for event handling.

Reply:
Ok then, either way works, you can keep how you have it.